tests/integration/spark/test_spark_config.py

```diff
@@ -45,56 +45,39 @@ def test_spark_config_datasource(spark_session_v012):
     old_app_id = spark_session.sparkContext.applicationId
-
-    if databricks_runtime:
-        # simulate a databricks runtime environment by setting the databricks runtime version
-        os.environ["DATABRICKS_RUNTIME_VERSION"] = "7.3"
-
-    name: str = "great_expectations-ee-config"
-    spark_config: Dict[str, str] = {
-        "spark.app.name": name,
+    new_spark_config: Dict[str, str] = {
+        "spark.app.name": "great_expectations-ee-config",
         "spark.sql.catalogImplementation": "hive",
         "spark.executor.memory": "512m",
         # "spark.driver.allowMultipleContexts": "true",  # This directive does not appear to have any effect.
     }
-    execution_engine = SparkDFExecutionEngine(spark_config=spark_config)
-    spark_session: SparkSession = execution_engine.spark
+    execution_engine = SparkDFExecutionEngine(spark_config=new_spark_config)
+    new_spark_session: SparkSession = execution_engine.spark
+
     # noinspection PyProtectedMember
-    sc_stopped: bool = spark_session.sparkContext._jsc.sc().isStopped()
+    sc_stopped: bool = new_spark_session.sparkContext._jsc.sc().isStopped()
+
     assert not sc_stopped
 
-    # Test that our values were set if not running in a Databricks runtime
-    conf: List[tuple] = execution_engine.spark.sparkContext.getConf().getAll()
-    if not databricks_runtime:
-        # we can safely "restart" the SparkContext
-        if pyspark_version.major >= 3:
-            # spark context is restarted because the "spark.app.name" is not the one desired
-            # this happens because from Spark 3, the SparkSession builder won't update config values
-            assert old_app_id != execution_engine.spark.sparkContext.applicationId
-        assert ("spark.app.name", "great_expectations-ee-config") in conf
-        assert ("spark.sql.catalogImplementation", "hive") in conf
-        assert ("spark.executor.memory", "512m") in conf
-    else:
-        # we should not "restart" the SparkContext
-        if pyspark_version.major >= 3:
-            assert old_app_id == execution_engine.spark.sparkContext.applicationId
-            # spark context config values are not changed
-            assert (
-                "spark.app.name",
-                "default_great_expectations_spark_application",
-            ) in conf
-        else:
-            # spark context config values are changed by the builder no matter what
-            assert ("spark.app.name", "great_expectations-ee-config") in conf
-            assert ("spark.sql.catalogImplementation", "hive") in conf
-            assert ("spark.executor.memory", "512m") in conf
+    current_spark_config: List[
+        tuple
+    ] = execution_engine.spark.sparkContext.getConf().getAll()
+    assert old_app_id == execution_engine.spark.sparkContext.applicationId
+    assert ("spark.sql.catalogImplementation", "hive") in current_spark_config
+    assert (
+        "spark.app.name",
+        "default_great_expectations_spark_application",
```

Comment:
Is this the main thing that is being tested? That this name doesn't change to `great_expectations-ee-config`? Would you mind adding a docstring and comment to this test to explain a bit more about what it is doing and why?

Reply:
@anthonyburdi I will add the docstring.

Reply:
@anthonyburdi Committed